concrete_datastore/concrete/models.py

```diff
@@ -829,13 +828,10 @@ class Meta:
                 {'blank': True, 'null': True, 'validators': [validate_file]}
             )
         elif field.f_type == 'JSONField':
-            json_args = args
-            json_args['blank'] = True
-            json_args['encoder'] = None
-            json_args['null'] = False
-            json_args['default'] = dict
-            attrs.update({field_name: JSONField(**json_args)})
-            continue
+            args['blank'] = True
+            args['encoder'] = None
+            args['null'] = False
+            args['default'] = dict
```

Comment:
could you explain this change ?

Reply:
The previous JSONField is deprecated, we can use the default field from odels: https://docs.djangoproject.com/en/4.0/releases/3.1/#jsonfield-for-all-supported-database-backends